src/components/Tool/Appeal/Shared/handleReceivedSnackBarNotifications/handleReceivedSnackBarNotifications.ts

```diff
@@ -0,0 +1,49 @@
+import { TFunction } from 'i18next';
+import { OptionsObject } from 'notistack';
+import { Maybe, PledgeStatusEnum } from 'src/graphql/types.generated';
+import { AppealStatusEnum } from '../../AppealsContext/AppealsContext';
+
+interface HandleReceivedSnackBarNotificationsProps {
+  dbStatus: Maybe<PledgeStatusEnum> | undefined;
+  selectedAppealStatus: AppealStatusEnum;
+  t: TFunction;
+}
+const handleReceivedSnackBarNotifications = ({
```

Comment:
Instead of returning the args to pass to `enqueueSnackbar`, what about passing in `enqueueSnackbar` and letting this helper function call it directly?

Reply:
I did this initially but double-guessed what you would say and changed it to this. lol, I'll change it back.